src/Twig/Extension/FormatterMediaExtension.php

```diff
@@ -11,13 +11,14 @@
 
 namespace Sonata\MediaBundle\Twig\Extension;
 
-use Sonata\FormatterBundle\Extension\BaseProxyExtension;
 use Sonata\MediaBundle\Model\MediaInterface;
 use Sonata\MediaBundle\Twig\TokenParser\MediaTokenParser;
 use Sonata\MediaBundle\Twig\TokenParser\PathTokenParser;
 use Sonata\MediaBundle\Twig\TokenParser\ThumbnailTokenParser;
+use Twig\Extension\AbstractExtension;
+use Twig\Extension\ExtensionInterface;
 
-class FormatterMediaExtension extends BaseProxyExtension
```

Comment:
won't this be a BC break?

Reply:
Only if you test for `instance of Sonata\FormatterBundle\Extension\BaseProxyExtension`. The logic and implemented interfaces is completly moved to this repo.